pkg/backup/resource_backupper.go

```diff
@@ -129,7 +129,9 @@ func (rb *defaultResourceBackupper) backupResource(
 		// (all namespaces) backup. Note that in the case of a subset of
 		// namespaces being backed up, some related cluster-scoped resources
 		// may still be backed up if triggered by a custom action (e.g. PVC->PV).
-		if !resource.Namespaced && !rb.namespaces.IncludeEverything() {
+		// If we're processing namespaces themselves, we will not skip here, they may be
+		// filtered out later.
+		if !resource.Namespaced && resource.Kind != "Namespace" && !rb.namespaces.IncludeEverything() {
```

Comment:
This will skip if the resource is a namespace, but doesn't appear to filter based on included/excluded namespaces. Is that intentional?

Reply:
It was, until we realized that the itemBackupper's filtering of namespaces based on includes/excludes only applies to namespace-scoped resources, but it won't work for namespaces (they're cluster-scoped).

Reply:
Right, I saw that code earlier. Is there some sort of cluster-scoped filtering that could be introduced?

Reply:
This is only messy because it's the namespace object itself